src/Extend/Theme.php

```diff
@@ -52,6 +53,53 @@ public function overrideFileSource(string $file, string $newFilePath, string $ex
         return $this;
     }
 
+    /**
+     * This method allows you to add custom Less functions.
+     *
+     * **Example usage:**
+     * ```php
+     * (new Extend\Theme)
+     *     ->addCustomLessFunction('is-flarum', function (mixed $text) {
+     *         return new Less_Tree_Quoted('', strtolower($text) === 'flarum' ? 'true' : 'false')
+     *     }),
+     * ```
+     *
+     * @see https://leafo.net/lessphp/docs/#custom_functions
+     *
+     * @param string $functionName Name of the function identifier.
+     * @param callable $callable The PHP function to run when the Less function is called.
+     * @return self
+     */
+    public function addCustomLessFunction(string $functionName, callable $callable): self
+    {
+        if (! is_callable($callable)) {
+            return $this;
+        }
+
```

Comment:
Shouldn't this be impossible due to dynamic type checking?

Reply:
Yes...
```suggestion
```